README.md

```diff
@@ -1,82 +1,177 @@
-## Setup Guide
+1. Launch VS Code, in the Extension tab, find and install **"Azure IoT Device Workbench"**.
 
-### MXChip IoT DevKit
+    ![Extension install Device Workbench](./docs/images/device-workbench.png)
 
-Please follow the [guide](docs/iot-devkit.md) to setup IoT DevKit.
+2. Reload the window and make sure all the dependency extensions are correctly installed:
 
-Here are a set of tutorials that will help you get started:
+   - Azure Account
+   - Azure IoT Hub Toolkit
+   - C/C++
+   - IoT Device Cube
+   - Remote Development
 
-* [Get Started](./docs/iot-devkit/devkit-get-started.md)
-* [Shake, Shake for a Tweet](./docs/iot-devkit/devkit-shakeshake.md)
-* [Remote Monitoring](./docs/iot-devkit/devkit_remote_monitoringv2.md)
-* [DevKit Translator](./docs/iot-devkit/devkit-translator.md)
-* [MQTT Client](./docs/iot-devkit/devkit-mqtt-client.md)
-* [DevKit State](./docs/iot-devkit/devkit-state.md)
-* [Door Monitor](./docs/iot-devkit/devkit_door_monitor.md)
-* [DevKit DPS](./docs/iot-devkit/devkit_dps.md)
-* [DevKit OTA](./docs/iot-devkit/devkit-ota.md)
-* [Stream Analytics and Cosmos DB](./docs/iot-devkit/devkit-stream-analytics-cosmos-db.md)
+### Install Docker
 
-### teXXmo IoT button
+[VS Code Remote](https://aka.ms/vscode-remote) needs Docker runtime installed in order to use local containers, which is required for developing Embedded Linux devices in the Device Workbench.
 
-Here are a set of tutorials that will help you get started:
+- Install Docker Desktop for Windows
+
+  Follow the [installation guide](https://docs.docker.com/docker-for-windows/install/) and make sure to enable driver sharing for Docker to access your local folder:
+
+  1. Right click the **Docker** icon in the task try and select "Settings".
+  2. Click on the "Shared Drivers" section and choose the driver to be shared.
+
+     ![Shared driver](./docs/images/shared-drivers.png)
+
+- Install Docker Desktop for Mac
+
+  Follow the [installation guide](https://docs.docker.com/docker-for-mac/install/) and make sure to enable driver sharing for Docker to access your local folder:
+    *[TBD for Mac]*
+
+- Get Docker for Linux
+
+  Linux is a highly variable environment and the large number of server, container, and desktop distributions can make it difficult to know what is supported. Please check this [tutorial](https://code.visualstudio.com/docs/remote/linux) for Linux support.
+
+## Usage
+
+Depending on the device platform you choose, follow the resources below to develop on it.
+
+### Embedded Linux <sup>preview</sup>
+
+To simplify the cross compiling toolchain, Azure IoT Device SDK and dependencies setup and configuration, Device Workbench put all these components in the container. All cross-compiling works happen in it. Here is the quick start of using it to develop and compile a simple device app written in C and running on [Raspberry Pi 3 Model B+](https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/) to send telemetry data to Azure IoT Hub.
```

Comment:
The 32-bit Raspberry project does not require the device to be RPI3 Model B+. It is okay if the Raspberry Pi board can run 32-bit OS.

Reply:
Current the Raspbian is 32bit by default.